server/views/dashboard/_notificationBanner.njk

```diff
@@ -0,0 +1,17 @@
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+
+{% set html %}
+  <h3 class="govuk-notification-banner__heading">Rolling out the referral service</h3>
+  <p class="govuk-body">This is a new service. While we're rolling it out, referrers can only use it to refer people to Accredited Programmes at a few prisons. At the moment, you can refer to:</p>
```

Comment:
We've got `$govuk-global-styles` set to `true` so we don't need to add `govuk-body`.  Same applies for `<a>` and `govuk-link`.